### PR TITLE
Task/add attr metadata store conf option ckan

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix issue https://github.com/telefonicaid/fiware-cygnus/issues/1912

### CODE REVIEW

**Review by IvanHdzC — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,2 @@
-[cygnus-common][SQLBackendImpl] Exception handle for connection/statement SQL objects.
```
Comment (manucarrace) [inline CHANGES_NEXT_RELEASE:1]: This line should be deleted?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: That entry is about previous PR https://github.com/telefonicaid/fiware-cygnus/issues/1907, but is not deleted, just edited

**Comment by manucarrace** [comment]
LGTM

**Pull request merged**